en/components/grids_templates/toolbar.md

```diff
@@ -456,156 +456,104 @@ To get started with styling the toolbar, we need to import the index file, where
 @import '~igniteui-angular/lib/core/styles/themes/index';
 ```
 
-Following the simplest approach, we create a new theme that extends the [`igx-grid-toolbar-theme`]({environment:sassApiUrl}/index.html#function-igx-grid-toolbar-theme) and accepts the `$background-color` and the `$title-text-color` parameters.
+First, let's create a new palette.
 
 ```scss
-$dark-grid-toolbar-theme: igx-grid-toolbar-theme(
-    $background-color: #292826,
-    $title-text-color: #FFCD0F
+$my-dark-palette: igx-palette(
+    $primary: #2466ff,
+    $secondary: #FFCD0F,
+    $surface: #2a2b2f,
+    $grays: #fff,
 );
+
+$my-dark-color: igx-color($my-dark-palette, 'surface');
 ```
 
-In order to style the buttons inside the toolbar, we will also create another theme that extends the [`igx-button-theme`]({environment:sassApiUrl}/index.html#function-igx-button-theme).
+Now, create a new theme that extends the [`igx-grid-toolbar-theme`]({environment:sassApiUrl}/index.html#function-igx-grid-toolbar-theme) and modify the `$background-color` and the `$title-text-color` parameters.
 
 ```scss
-$dark-button-theme: igx-button-theme(
-    $outlined-background: #FFCD0F,
-    $outlined-text-color: #292826,
-    $outlined-hover-background: #404040,
-    $outlined-hover-text-color: #FFCD0F
+$dark-grid-toolbar-theme: igx-grid-toolbar-theme(
+    $palette: $my-dark-palette,
+    $background-color: $my-dark-color,
+    $title-text-color: igx-color($my-dark-palette, 'secondary'),
+    $dropdown-background: $my-dark-color,
 );
 ```
 
-The last step is to **include** the newly created themes. The button theme will be scoped to the actions container of the toolbar, so the buttons outside the toolbar do not get affected by it.
+To theme the column actions menus of the toolbar we have to change the theme for the [`igx-column-actions-theme`]({environment:sassApiUrl}/index.html#function-igx-column-actionsr-theme) component.
 
 ```scss
-@include igx-grid-toolbar($dark-grid-toolbar-theme);
-.igx-grid-toolbar__actions {
-    @include igx-button($dark-button-theme);
-
-    .igx-button--outlined {
-        margin-left: 0.5rem;
-        border: none;
-    }
-}
+$dark-column-actions-theme: igx-column-actions-theme(
+    $palette: $my-dark-palette,
+    $title-color: igx-color($my-dark-palette, 'secondary'),
+    $background-color: igx-color($my-dark-palette, 'surface')
+);
 ```
 
->[!NOTE]
->If the component is using an [`Emulated`](../themes/component-themes.md#view-encapsulation) ViewEncapsulation, it is necessary to `penetrate` this encapsulation using `::ng-deep`:
+Since the column actions are using other components, igx-button, igx-checkbox, and igx-input-group, we need to change their themes to match our new toolbar theme.
```

Comment:
```suggestion
Since the column actions are using other components - igx-button, igx-checkbox, and igx-input-group, we need to change their themes to match our new toolbar theme.
```